### PR TITLE
Update docs and advise user to use higher nginx timeout

### PR DESCRIPTION
**Pull request by 1-alex98**
Only doc changes. Advise user to use higher nginx timeout.

### CODE REVIEW

File: docs/dev/setup/jenkins-gitlab.rst [hunk 1]
```diff
@@ -385,8 +385,47 @@ Jenkins Server Setup
    ::
 
        echo "client_max_body_size 16m;" > client_max_body_size.conf
+5. The NGINX default timeout is pretty low. For plagarism check and unlocking student repos for the exam a higher timeout is advisable. Therefore we write our own nginx.conf and load it in the container.
+   ::
+
+       echo "
```
Comment (1-alex98) [inline docs/dev/setup/jenkins-gitlab.rst:391]: Does anybody know how I would escape the content here or any idea how to better set the timeout in nginx? :thinking:

**Comment by krusche** [comment]
@ivanchimeno can you please take over this PR? it is related to Jenkins and Gitlab

**Review by ivanchimeno — APPROVED** [review]
Approving because the timeout changes seem reasonable :+1:

**Review by FrankeLukas — APPROVED** [review]
Can't comment on the technical details, but the docs look good to me.

**Comment by 1-alex98** [comment]
Many requests take really long and therefore time out. So things break with a too low timeout. We tried that was the cause for at least 2 errors we had.

**Pull request merged**